settings.gradle

```diff
@@ -39,6 +39,8 @@ dependencyResolutionManagement {
             library('bouncycastle-bcpkix', 'org.bouncycastle', 'bcpkix-jdk18on').versionRef('bouncycastle')
             version('guava', '32.0.1-jre')
             library('guava-core', 'com.google.guava', 'guava').versionRef('guava')
+            version('reflections', '0.9.12')
```

Comment:
This downgrade is still a problem. What options do we have to support the latest version?